src/components/ha-form/ha-form.ts

```diff
@@ -151,40 +144,19 @@ export class HaForm extends LitElement implements HaFormElement {
             </div>
           `
         : ""}
-      <div id="element"></div>
+      ${dynamicContentDirective(
+        `ha-form-${this.schema.type}`,
+        {
+          schema: this.schema,
+          data: this.data,
+          label: this._computeLabel(this.schema),
+          suffix: this._computeSuffix(this.schema),
+        },
+        { id: "child-form" }
```

Comment:
You can actually set IDs as a property. So no reason yet to add attributes to set.